work/fourth/index.html

```diff
@@ -56,6 +56,20 @@
       // .slideUp() 要素がスライドしながら非表示されます。
       // .children() 要素の子要素を取得します。
       // .stop() 現在動作中のアニメーション処理を全て中止します。
+      $(function() {
+        $('.menu__list').children('li').hover(slideDown,slideUp);
```

Comment:
[nits]
よく書けていますね！
とても細かいですが、メソッドを使用するより下記の方が直感的で記述量も減りおすすめです！

```suggestion
        $('.menu__list > li').hover(slideDown,slideUp);
```